apps/backend/src/users/users.controller.ts

```diff
@@ -23,6 +24,12 @@ export class UsersController {
     return await this.usersService.findAll();
   }
 
+  @Get(':id/bac')
+  @ApiOperation({ summary: 'Get BAC (Blood Alcohol Content)' })
+  async calBac(@Param('id', ParseUUIDPipe) id: string): Promise<UserBac> {
+    return await this.usersService.calBac(id);
+  }
```

Comment:
Now that authentication is set up, we can remove the `:id` from the route and use the userData from the jwt in the service method